input/write_reactors.py

```diff
@@ -23,69 +25,123 @@ def write_reactors(csv_file, reactor_template, region_template, reactor_output,
 
     """
 
-    # display usage if in error
-    if len(sys.argv) <3:
-        print("Usage: 'Python write_reactors.py [csv] [reactor_template] [region_template]\
-             [reactor_output] [region_output]")
 
-    reactor_lists= np.genfromtxt(csv_file,
-                                delimiter=',',
-                                dtype=('S128','S128', 'int','int', 'int'),
-                                names=('country','reactor_name', 'capacity',
-                                       'n_assem_core','n_assem_batch'))
+def delete_file(file):
+    """
+    This function checks if there is a file with the same name,
+    and deletes the file if it exists.
+    """
+    if os.path.exists('./' + file) is True:
+        os.system('rm ' + file)
+
+
+def read_csv(csv_file):
+    """
+    This function reads the csv file and returns the array.
+
+    input)
+        csv_file: csv file that lists country, reactor name,
+                 capacity, #assem per core, # assem per batch.
+
+    return)
+        reactor_lists: array with the data from csv file
+    """
+
+    reactor_lists = np.genfromtxt(csv_file,
+                                 delimiter=',',
+                                 dtype=('S128', 'S128', 'int', 'int', 'int'),
+                                 names=('country', 'reactor_name', 'capacity',
+                                       'n_assem_core', 'n_assem_batch'))
+
+    return reactor_lists
+
+
+def read_template(template):
+    """
+    This function takes a jinja template file and returns it
+
+    input)
```

Comment:
I would strongly recommend using a standardized docstring format. An example is the numpy sphinx docstring format: 

```python

def function_with_types_in_docstring(param1, param2):
    """Example function with types documented in the docstring.

    `PEP 484`_ type annotations are supported. If attribute, parameter, and
    return types are annotated according to `PEP 484`_, they do not need to be
    included in the docstring:

    Parameters
    ----------
    param1 : int
        The first parameter.
    param2 : str
        The second parameter.

    Returns
    -------
    bool
        True if successful, False otherwise.

    .. _PEP 484:
        https://www.python.org/dev/peps/pep-0484/

    """

```